.github/workflows/php-cs-fixer.yml

```diff
@@ -11,7 +11,7 @@ jobs:
                 uses: actions/checkout@v1
 
             -   name: Fix style
-                uses: docker://oskarstark/php-cs-fixer-ga
+                uses: docker://oskarstark/php-cs-fixer-ga:2.18.6
```

Comment:
https://github.com/OskarStark/php-cs-fixer-ga/issues/36

Not really sure what the progress on that will be. This seems to break every other PR as of today...

Reply:
@wolfrednicolas is going through and fixing those for us.